pulp_2to3_migration/app/plugin/docker/migrator.py

```diff
@@ -129,6 +129,7 @@ def pipeline_stages(self):
             DockerContentSaver(),
             ResolveContentFutures(),
             InterrelateContent(),
+            #RemoveEmtpyTags(),
```

Comment:
leaving post_hook only would not solve problem for upgraded installations that already contain problem with `tagged_manifest=null`. In this case a migration is needed to remove existing tags that point to null manifest andd then leave all the subsequent job for future runs on the post_save hook.
or - we get implemented this stage the would loop trough dc and check type(dc.content)  ==  Tag and remove those that have `tagged_manifest=null`.